content/en/docs/concepts/storage/mount-propagation.md

```diff
@@ -0,0 +1,77 @@
+---
+reviewers:
+- saad-ali
+- jsafrane
+- thockin
+- msau42
+title: Mount Propagation
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Mount propagation allows for sharing volumes mounted by a container to
+other containers in the same pod, or even to other pods on the same node.
```

Comment:
Consider using glossary tooltips here for at least volume, and possibly also container, pod and node.

Reply:
✔